ocaml/xapi-storage/generator/lib/data.ml

```diff
@@ -132,6 +140,23 @@ module Datapath (R : RPC) = struct
       ]
       (dbg @-> uri_p @-> domain @-> returning unit error)
 
+  let import_activate =
+    declare "import_activate"
+      [
+        "[import_activate uri domain] prepares a connection to the "
+      ; " storage named by [uri] for use by inbound import mirroring, "
+      ; "the [domain] parameter identifies which domain to connect to, "
+      ; "most likely 0 or a custom storage domain. The return value is a "
+      ; "path to a UNIX domain socket to which an open file descriptor "
+      ; "may be passed, by SCM_RIGHTS. This, in turn, will become "
```

Comment:
what is `SCM_RIGHTS`?

Reply:
See https://man7.org/linux/man-pages/man7/unix.7.html

Reply:
It's the mechanism by which a process can pass an open file descriptor to a separate process over a UNIX domain socket.